source/subs_functions/subs_functions.bas

```diff
@@ -1427,6 +1427,55 @@ id.ret = LONGTYPE - ISPOINTER
 id.hr_syntax = "_PRINTMODE[(imageHandle&)]"
 regid
 
+' a740g: Extended Font support
+clearid
+id.n = qb64prefix$ + "UFontHeight"
+id.Dependency = DEPENDENCY_LOADFONT
+id.subfunc = 1
+id.callname = "func__UFontHeight"
+id.args = 1
+id.arg = MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "[?]"
+id.ret = LONGTYPE - ISPOINTER
+id.hr_syntax = "_UFONTHEIGHT&[(fontHandle&)]"
+regid
+
+clearid
+id.n = qb64prefix$ + "UPrintWidth"
+id.Dependency = DEPENDENCY_LOADFONT
+id.subfunc = 1
+id.callname = "func__UPrintWidth"
+id.args = 3
+id.arg = MKL$(STRINGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "?[,[?][,[?]]]"

```

Comment:
Same thing here, the last parameter probably shouldn't be optional, and the `hr_syntax` should match.

Reply:
The last two parameters are optional. Same reason for `hr_syntax` as above.